packages/IMAPClient-Core.package/ICEmail.class/instance/deleteEmailFromDisk.st

```diff
@@ -2,4 +2,4 @@ action
 deleteEmailFromDisk
 
 	self folder removeMail: self. 
-	FileDirectory default deleteFileNamed: (self maildirFileName, FileDirectory slash, self uniqueID)		
\ No newline at end of file
+	ICFileReadWriter new delete: self
```

Comment:
Why do we need a new instance here? 
Couldn't that be static?

Reply:
You mean the Mail should hold a reference to a `ICFileReadWriter`. Putting this on the class side shouldn't be possible as this need access to the `endpoint maildirFilename` which for the mean time `ICEmail` offers as legacy API, but we will remove this shortly after. Then you will need to pass an `ICEndpoint` to create an `ICFileReadWriter`.

Reply:
Just noticed that closing #192 requires to remove `maildirFilename` so I will change this.